.buildkite/pipeline.yml

```diff
@@ -0,0 +1,38 @@
+# SPDX-FileCopyrightText: 2021 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+steps:
+  - label: REUSE lint
+    commands:
+      - nix-build -A checks.x86_64-linux.reuse-lint
+
+  - label: Check trailing whitespaces
+    commands:
+      - nix-build -A checks.x86_64-linux.trailing-whitespace
+
+  - label: files generated by stack2cabal are up to date
+    commands:
+      - nix-shell --run "stack2cabal"
+      - git diff --exit-code
+
+  - label: build
+    commands:
+      - nix-build -A checks.x86_64-linux.build-all
+
+  - label: hlint
+    commands:
+      - nix run -f. legacyPackages.x86_64-linux.haskellPackages.hlint
+        -c hlint -j --hint .hlint.yaml */
+
+  - label: weeder
+    commands:
+      - nix-build -A weeder-script
+      - ./result
+
+  # don't run tests until all components are built
+  - wait
+
+  - label: tests
+    commands:
+      - nix-build -A checks.x86_64-linux.test
```

Comment:
Is this actually running the tests or just building them?

I would have expected to see the test names in the logs, but there's nothing there: https://buildkite.com/serokell/haskell-algorand-sdk/builds/22#8f0e2142-0f2c-4508-a7b7-5b66ad70e6ec

Reply:
hm....
https://buildkite.com/serokell/haskell-algorand-sdk/builds/14#7d983bf3-2ead-4f22-9402-177162c8f2b4

here we can see test names

Reply:
@balsoft we cannot see tests logs at last CI launch because of caching?

Reply:
Yes, that's correct. Nix doesn't do stuff it has already done once.